components/Ui/UiCard.vue

```diff
@@ -44,16 +44,13 @@
               {{ tag }}
             </bx-tag>
           </div>
-          <div v-if="hasTags(tooltipTags)" class="card__tags">
+          <div v-if="hasTags(primaryTags)" class="card__tags">
             <div
-              v-for="tag in tooltipTags"
+              v-for="tag in primaryTags"
               :key="tag.label"
               class="card__custom-pill"
             >
               {{ tag.label }}
-              <bx-tooltip-icon :body-text="tag.description" direction="bottom">
-                <Information16 class="card__tooltip__icon" />
-              </bx-tooltip-icon>
             </div>
           </div>
```

Comment:
We will have only one primary tag, so I think we can rewrite this with something like this:

```
<bx-tag
    v-if="primaryTag"
    class="card__tag"
    type="purple"
  >
    {{ primaryTag }}
</bx-tag>
```

Please, review styles, this is just an example :D